include/tlog/sink_type.h

```diff
@@ -109,6 +118,7 @@ struct tlog_sink_type {
     tlog_sink_type_write_fn     write;      /**< Writing function */
     tlog_sink_type_cut_fn       cut;        /**< I/O-cutting function */
     tlog_sink_type_flush_fn     flush;      /**< Flushing function */
+    tlog_sink_type_close_fn     close;      /**< Close function */
```

Comment:
I think this should be called `io_close` and `I/O-closing function` to match everything else.